code/modules/admin/verbs/listahelps.dm

```diff
@@ -0,0 +1,307 @@
+/datum/adminticket
+	var/ID = "" //ID of the ticket, very important as its used to find adminhelps.
+	var/user = "" //The user of the ahelp.
+	var/uckey //The saved ckey of the adminheloing user.
+	var/admin = "N/A" //The handling admin? Like come on.
+	var/msg = "" //The adminhelp message.
+	var/resolved = "No" //Is it resolved? Its much easier to have a "Yes" or a "No", as you can directly concat it into strings making life that much easier.
```

Comment:
what i would suggest is the following.

`var/status` on a ticket

then the following few defines

```
#DEFINE TICKET_STATUS_RESOLVED 1
#DEFINE TICKET_STATUS_UNRESOLVED 2
#DEFINE TICKET_STATUS_ASSIGNED 4
```

and then setting `status = TICKET_STATUS_ASSIGNED` instead of using a string there, you could also just have two variables on the ticket! 👍